.github/workflows/stechec2-generator-test.yml

```diff
@@ -14,11 +14,15 @@ jobs:
       - uses: actions/checkout@v2
 
       - name: Install generator dependencies
+        # Some packages here are included in the --ldflags of php-config but
+        # not present by default in the CI, so we install them manually.
+        # (libargon2-1 and libsodium-dev)
         run: |
           sudo apt-get update
           sudo apt-get install \
             python3 python3-dev python3-yaml python3-jinja2 flake8 php-cli \
-            php-dev libphp-embed ocaml mono-devel ghc openjdk-8-jdk
+            php-dev libphp-embed ocaml mono-devel ghc openjdk-8-jdk \
+            libargon2-0-dev libsodium-dev  # XXX: to remove
```

Comment:
Why to remove if we need them for php-config?

Reply:
Yeah it's not really "to remove", it's more like "file upstream/distribution bugs because these aren't included and then remove"